CHITTY.md

```
@@ -2,17 +2,366 @@
+GET  /api/workflows/stats             → WorkflowStatsResult
+POST /api/workflows                   → WorkflowRecord (201)
+PATCH /api/workflows/:id              Body: { status?, duration? }
+POST /api/chittyflow/workflows        Body: { appId, workflows[] }
+                                      → { synced: N, workflows: [] }
+```
+
+---
+
+### Health Monitoring
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)